docs/user-guide/installation/index.md

```diff
@@ -17,27 +17,74 @@ The first step to using HELICS is to install it. Since there are several ways to
 
 As of HELICS v3, the only supported language bindings that are included with the core HELICS library downloads are C and C++98, in addition to C++17 when building from source. If you end up needing to build from source AND use one of the supported language bindings you'll need to follow the instructions for installing HELICS for said language. This would also be the case if you were needing to run a co-simulation that used tools that provided their HELICS implementation in a variety of languages. Generally speaking, as long as all supported languages are on similar versions, each one can use it own installed version of HELICS without any trouble. The supported languages also have ways of being pointed towards a specific HELICS installation (rather than the one they install for themselves) if that is preferred or necessary for a particular use case.
 
-![](https://github.com/GMLC-TDC/helics_doc_resources/raw/main/user_guide/install-decision-tree.png)
+```{mermaid}
+%%{init: {'theme': 'neutral'}}%%
+graph LR
+    Start(Do you want to install a <br/> standard HELICS version?) -->|Yes| languageChoice
+    Start -->|No| C
+    subgraph Source Build
+        C[Build from source for the <br> particular branch or <br/> configuration that you need.]
```

Comment:
Maybe this could also be a link to https://docs.helics.org/en/latest/user-guide/installation/build_from_source.html